hashicorp/dev-portal
# refactor: update integrations list items

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-ksadjust-integrations-with-glob-c430dd-hashicorp.vercel.app/waypoint/integrations) 🔎
- [Asana task](https://app.asana.com/0/1199634971449915/1204121282604743) 🎟️

## 🗒️ What

This PR updates two integrations components, the parameter list item on the components view and the 'view details' link on the integrations landing list view. 

### Production
<img width="973" alt="Screenshot 2023-03-08 at 12 18 04 PM" src="https://user-images.githubusercontent.com/36613477/223840354-67e91786-4bca-4950-9fea-5fe2a7ade840.png">

<img width="972" alt="Screenshot 2023-03-08 at 12 18 12 PM" src="https://user-images.githubusercontent.com/36613477/223840380-299247dd-2a5c-4ff1-b8da-d02acce3bed8.png">

### Preview

<img width="991" alt="Screenshot 2023-03-08 at 12 19 33 PM" src="https://user-images.githubusercontent.com/36613477/223840578-1d27fecd-260c-4ced-9d15-b92bcf3f5409.png">
<img width="970" alt="Screenshot 2023-03-08 at 12 19 40 PM" src="https://user-images.githubusercontent.com/36613477/223840580-791ea791-0aac-4264-8cb5-d55c093cd7a0.png">


## 🧪 Testing

- Visit an[ integrations landing page](https://dev-portal-git-ksadjust-integrations-with-glob-c430dd-hashicorp.vercel.app/waypoint/integrations) with dark mode enabled, validate that the 'view details' link has the right color and retains the same positioning as production
- Visit a [component view page](https://dev-portal-git-ksadjust-integrations-with-glob-c430dd-hashicorp.vercel.app/waypoint/integrations/hashicorp/aws-alb/latest/components/release-manager), validate that the parameter list 'cards' use the same value as the cards across the rest of the app (see[ integrations landing page list](https://dev-portal-git-ksadjust-integrations-with-glob-c430dd-hashicorp.vercel.app/waypoint/integrations) as an example)

## 💭 Anything else?
